app/workers/delete_service_hierarchy_worker.rb

```diff
@@ -7,11 +7,11 @@ class DeleteServiceHierarchyWorker < DeleteObjectHierarchyWorker
 
   # TODO: when we remove the Rolling Update, we must remove this whole method
   def destroy_and_delete_associations
-    super
-
-    return unless service.account.provider_can_use?(:api_as_product)
+    unless service.account.provider_can_use?(:api_as_product)
```

Comment:
👍 Thank you. Sadly that's the price of having rolling updates

Reply:
Indeed :+1: Thankfully, we should remove this whole method soon when we get rid of this RU :smile: